docs/installation/installation-guide.md

```diff
@@ -2,7 +2,7 @@
 
 This document will provide you with information about 2 ways of developing and runninng the Shopsys Framework project and the services that it depends on.  
 The first option [using docker](#installation-using-docker) is **highly recommended** since it is the easiest and fastest way to start Shopsys Framework.
-In the case the operation system does not support docker, we prepared also second section with document about project [installation without docker](#installation-without-docker), however this way is slower and harder to configure and maintain because of different operation systems and their versions.
+In the case the operation system does not support docker, or you are not able to use Docker for some reason, we prepared also second section with document about project [installation without docker](#installation-without-docker), however this way is slower and harder to configure and maintain because of different operation systems and their versions.
```

Comment:
this seems to be a bit strange. please try to describe the real problem why would the developer want to not use it.

Reply:
I tried to rewrite it, is it better now?